pkg/idp/util.go

```diff
@@ -60,6 +60,15 @@ func getBaseDomainFromClusterValues(clusterValuesConfigmap *corev1.ConfigMap) st
 	return ""
 }
 
+func connectorsDefinedInUserConfigMap(userConfigmap *corev1.ConfigMap) bool {
+	values := userConfigmap.Data[key.UserValuesConfigMapKey]
+	rex := regexp.MustCompile(fmt.Sprintf(`(%v)(\s*:\s*)(\S+)`, key.ConnectorsKey))
```

Comment:
Would it be possible to parse the yaml data and look for the `connectors` key directly, instead of using regex?

Consider this madeup, semi-realistic test example:
```go
{
	name: "case 8",
	data: map[string]string{
		key.UserValuesConfigMapKey: `
			url: "connectors:8000"
			`,
	},
	expectedResult: false,
},
```

Reply:
Sounds like it might be possible but won't we need the values schema?
I don't really want to risk throwing an error here if the configmap has a weird format.

Reply:
Yeah, that might overcomplicate things.